man/computeVSIF.Rd

```diff
@@ -0,0 +1,87 @@
+\name{computeVSIF}
+\alias{computeVSIF}
+\alias{computeVSIFNullModel}
+\title{
+Computes variant-specific inflation factors
+}
+\description{
+Computes variant-specific inflation factors resulting from differences 
+in variances and allele frequencies across groups pooled together in analysis.
+}
+\usage{
+computeVSIF(eafs, ns, sigma_sqs)
+computeVSIFNullModel(null.model, eafs, group_var_vec)
+}
+\arguments{
+  \item{eafs}{A named vector of a matrix of effect allele frequencies across
+  groups. Vector/column names are group names.}
```

Comment:
does it *have to* be effect allele frequencies? presumably it could be ref or alt allele freq just as well. is there another name that is more consistent with other GENESIS functions?

Reply:
renamed to `freq` for consistency with other functions